docs/manuals/source/IntroductionAndTutorial/Tutorial.rst

```diff
@@ -275,77 +350,125 @@ you should get the following output:
+Here, Bareos has listed the three different Jobs that you can run, and you should choose
+number :strong:`1` and type enter, at which point you will get:
 
 
 
 .. code-block:: bconsole
    :caption: run job
 
    Run Backup job
-   JobName:  BackupClient1
+   JobName:  backup-bareos-fd
    Level:    Incremental
    Client:   bareos-fd
    Format:   Native
    FileSet:  SelfTest
-   Pool:     Full (From Job resource)
-   NextPool: *None* (From unknown source)
+   Pool:     Incremental (From Job IncPool override)
    Storage:  File (From Job resource)
-   When:     2013-05-23 14:50:04
+   When:     2023-09-23 12:54:44
    Priority: 10
    OK to run? (yes/mod/no):
 
 
 
-At this point, take some time to look carefully at what is printed and understand it. It is asking you if it is OK to run a job named :config:option:`dir/job = BackupClient1`\  with FileSet :config:option:`dir/fileset = SelfTest`\  as an Incremental job on your Client, and to use Storage :config:option:`Dir/Storage = File`\  and Pool :config:option:`dir/pool = Full`\ , and finally, it wants to run it now (the current time should be displayed by your console).
+At this point, take some time to look carefully at what is printed and understand it.
+It is asking you if it is OK to run a job named :config:option:`dir/job = BackupClient1`\
+with FileSet :config:option:`dir/fileset = SelfTest`\ as an Incremental job on your
+Client, and to use Storage :config:option:`Dir/Storage = File`\ and Pool
+:config:option:`dir/pool = Incremental`\ ,and finally, it wants to run it now (the
+current time should be displayed by your console).
+
+Here we have the choice to run (:strong:`yes`), to modify one or more of the above
+parameters (:strong:`mod`), or to not run the job (:strong:`no`). Please enter :strong:`yes`,
+at which point you should immediately get a job queued confirmation and the associated
+jobid and the command prompt (an asterisk).
 
-Here we have the choice to run (yes), to modify one or more of the above parameters (mod), or to not run the job (no). Please enter yes, at which point you should immediately get the command prompt (an asterisk).
+.. code-block:: bconsole
+   :caption: job queued
+
+   Job queued. JobId=1
+   You have messages.
+   *
 
-If you wait a few seconds, then enter the command :bcommand:`messages` you will get back something like:
+If you wait a few seconds, then enter the command :bcommand:`messages` you will get back
+something like:
 
-.. TODO: Replace bconsole output by current version of Bareos.
 
 .. code-block:: bconsole
-   :caption: run
+   :caption: joblog output
 
    *<input>messages</input>
-   28-Apr-2003 14:30 bareos-sd: Wrote label to prelabeled Volume
-      "TestVolume001" on device /var/lib/bareos/storage
-   28-Apr-2003 14:30 rufus-dir: Bareos 1.30 (28Apr03): 28-Apr-2003 14:30
+   21-Sep 12:58 bareos-dir JobId 1: No prior Full backup Job record found.
```

Comment:
We have 21-Sep everywhere here, but the End time is 23-Sep.

Reply:
ok that's a mistake should be 23-Sep my regexp missed that one format. Will be so easier with unified timestamps ;-)